### PR TITLE
Fix SIGSEGV on pipelinerun logs

### PR DESCRIPTION
**Pull request by 16yuki0702**
# Changes
Related issue is https://github.com/tektoncd/cli/issues/775

When user defines embedded pipeline in pipelinerun,
```pr.Spec.PipelineRef``` is nil so SIGSEGV happen.

changing to Checking both ```pr.Spec.PipelineRef``` and ```pr.Spec.PipelineSpec```,
And getting tasks from either.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: pkg/log/pipeline_reader.go [hunk 1]
```diff
@@ -194,6 +191,28 @@ func (r *Reader) setUpTask(taskNumber int, tr trh.Run) {
...
+	case pr.Spec.PipelineRef != nil:
+		pl, err := r.clients.Tekton.TektonV1alpha1().Pipelines(r.ns).Get(pr.Spec.PipelineRef.Name, metav1.GetOptions{})
+		if err != nil {
+			return nil, err
+		}
+		tasks = pl.Spec.Tasks
+	case pr.Spec.PipelineSpec != nil:
+		tasks = pr.Spec.PipelineSpec.Tasks
+	default:
+		return nil, fmt.Errorf("pipelineRun %s not providing PipelineRef or PipelineSpec", pr.Name)
```
Comment (danielhelfand) [inline pkg/log/pipeline_reader.go]: nit: fmt.Errorf("PipelineRun %s did not provide PipelineRef or PipelineSpec", pr.Name)

Comment (16yuki0702) [reply pkg/log/pipeline_reader.go]: Thank for review !
It's based on this document :pray:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Comment (danielhelfand) [reply pkg/log/pipeline_reader.go]: Yes, my fault, agree on the error not being capitalized, but could you change to `fmt.Errorf("pipelinerun %s did not provide PipelineRef or PipelineSpec", pr.Name)` then?

Comment (16yuki0702) [reply pkg/log/pipeline_reader.go]: Oops, sorry I overlooked that...
Fixed it !!

**Pull request merged**